File: src/Services/ProfileService/Profile.API/DTOs/CompanyProfileDto.cs
namespace Profile.API.DTOs
{
    public class CompanyProfileDto
    {
        public Guid Id { get; set; }
        public string UserId { get; set; } = string.Empty;
        public string CompanyName { get; set; } = string.Empty;
        public string Description { get; set; } = string.Empty;
        public string Location { get; set; } = string.Empty;
        public string ContactEmail { get; set; } = string.Empty;
        public string ContactPhone { get; set; } = string.Empty;
        public string? WebsiteUrl { get; set; }
        public string? LinkedInUrl { get; set; }
        public string? LogoUrl { get; set; }
    }
}

File: src/Web/WebSPA/src/app/core/models/company-profile-dto.ts
export interface CompanyProfileDto {
  id: string,
  userId: string,
  companyName: string,
  description: string,
  location: string,
  contactEmail: string,
  contactPhone: string,
  websiteUrl?: string | null;
  linkedInUrl?: string | null;
  logoUrl?: string | null;
}
